DuckDuckGo/Tab/Navigation/DuckURLSchemeHandler.swift

```diff
@@ -291,11 +291,19 @@ private extension DuckURLSchemeHandler {
         guard let data = try? Data(contentsOf: URL(fileURLWithPath: file)) else {
             return nil
         }
-        let response = URLResponse(url: url, mimeType: mimeType(for: fileExtension), expectedContentLength: data.count, textEncodingName: nil)
+
+        let headerFields: [String: String] = [
+            "Content-type": mimeType(for: fileExtension),
+            "Content-length": String(data.count)
+        ]
+        guard let response = HTTPURLResponse(url: url, statusCode: 200, httpVersion: "HTTP/1.1", headerFields: headerFields) else {
```

Comment:
This change ensures that a successful response returns a HTTP 200 status code (to comply with Frontend lottie library requirements).